webapp/calendars/migrations/0020_auto_20151104_1112.py

```diff
@@ -13,10 +13,6 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RemoveField(
```

Comment:
Ok - tu jest ta zmaina :D zastanawiam się czy nie dać warunek - czy pole istnieje to skasuj? Chociaż to już jest bardziej dla sztuki.